src/ca11y/index.js

```diff
@@ -95,6 +95,9 @@ class Ca11y {
    * @return { Void }
    **/
   setDate(date, silent=false) {
+
+    if(date.toString() === "Invalid Date") return
```

Comment:
Are you sure this message is exactly the same on all platforms? In all locales? In the future? In older IE's it looks like an invalid date will have `NaN` returned from the `toString()` method - probably not a big deal for this library but worth noting.